### PR TITLE
create wiki and example of a file

### PR DESCRIPTION
**Pull request by solm11**

### CODE REVIEW

File: wiki/1/1.3.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+﻿
+##### 1.3 **סמלים לאומיים**
```
Comment (IgalKolihman) [inline wiki/1/1.3.md:2]: Stuff like this better be formatted to HTML tags (<h#>)

File: wiki/1/1.3.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
+﻿
```
Comment (IgalKolihman) [inline wiki/1/1.3.md:1]: This whole file formatted weirdly, comparing to the previous two.

Format it again, make sure that titles are not represented with #, but with <h#>

**Review by IgalKolihman — CHANGES_REQUESTED** [review]
First review, first workflow insights

File: wiki/1/1.2.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+•     מערכת פורמלית-מוסדית הכוללת תהליכים מוסכמים הקיימים בכל משטר דמוקרטי ובאמצעותם מתממש עקרון ריבונות העם: הכרעת הרוב, קיומה של אופוזיציה ובחירות חופשיות ושוויוניות.
+</p>
+
+<p dir="rtl">
+•     מערכת ערכית-תרבותית הכוללת ערכים דמוקרטיים בסיסיים, כגון: שמירה על כבוד האדם, שמירה על זכויות אדם בכלל ועל זכויות מיעוטים בפרט, קבלת תוצאותיהם של התהליכים הפורמליים
+(תוצאות הבחירות, מדיניות הממשלה הנבחרת) והשאיפה לחירות ולשוויון.
+</p>
+
+<p dir="rtl">
+במדינת ישראל מתקיימת הן המערכת הפורמלית (מוסדות שלטון דמוקרטיים, בחירות, חילופי שלטון) והן המערכת התרבותית (שוויון, אפליה מתקנת, מדיניות רווחה) . אחד מן ההיבטים הפורמליים החשובים ביותר הוא עקרון הפרדת הרשויות. משמעו, שהמשטר הדמוקרטי מורכב משלוש רשויות נפרדות: הרשות המחוקקת האחראית על חקיקתם של חוקים; הרשות המבצעת האחראית על ביצוע החוק ועל ניהול מוסדות המדינה; והרשות השופטת המפקחת על ביצוע החוק ונותנת סעד לאזרח אל מול החוק ואל מולם של אזרחים אחרים. כאשר שלושת הרשויות נפרדות, הן מאזנות ובולמות האחת את השנייה והכוח השלטוני מבוזר בין שלושתן. מצב בו אין הפרדת רשויות, והחוק ,המשפט והביצוע מרוכזים בידי אדם אחד, עשוי להביא לשחיתות שלטונית, לפגיעה בחירות, לאי-צדק ,למצב בו קבוצה קטנה משתלטת על עמדות הכוח במדינה או למצב
```
[the remark this answers is not in the document]
Comment (solm11) [reply wiki/1/1.2.md:21]: sounds great! :)

File: wiki/1/1.2.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+בו לא נשמרות זכויות המיעוט.
+</p>
+
+<h2>
+<p dir="rtl">
+1.2.1 הרשות המחוקקת: כנסת ישראל</p>
+</h2>
+
+<p dir="rtl">
+בישראל ,הכנסת מהווה את הרשות המחוקקת של המדינה .מכהנים בה 120 חברים, הנבחרים על־ידי הציבור אחת לארבע שנים (בהנחה שהבחירות אינן מוקדמות.) מקור השם "כנסת" הוא בכנסת הגדולה, אשר התכנסה בירושלים בימי הבית השני. בישראל שיטת הבחירות היא פרלמנטרית, הציבור בוחר במפלגות, המפלגות חוברות ביניהן על-מנת להקים את הממשלה וראש
```
[the remark this answers is not in the document]
Comment (solm11) [reply wiki/1/1.2.md:31]: dont understand the problem.

File: wiki/1/1.1.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
+﻿<h1>
+<p dir="rtl">
+1.1 פתיחה: מדינת ישראל כמדינה יהודית ודמוקרטית
+</p>
+</h1>
+
+<p dir="rtl">מייסדיה של ישראל שאפו להקים מדינה המקיימת תרבות ומשטר דמוקרטי ואשר אופייה ותרבותה הלאומיים יהיו מושתתים על היהדות. הם לא ראו בשני עקרונות אלו, עקרונות סותרים; אדרבה, על- פי מגילת העצמאות הם נובעים זה מזה: "מדינת ישראל... תהא מושתתת על יסודות החירות, הצדק והשלום – לאור חזונם של נביאי ישראל". סקרי דעת קהל ומחקרים מלמדים שהגדרה זו מקובלת על רובו המכריע של הציבור הישראלי, אולם לצד הסכמה זו מתקיים בישראל דיון ארוך שנים כיצד על היסודות היהודיים והדמוקרטיים לבוא לידי ביטוי.
+</p>
```
Comment (IgalKolihman) [inline wiki/1/1.1.md:8]: you didn't separate the paragraph content from the `<p>` tags
```suggestion
<p dir="rtl">
מייסדיה של ישראל שאפו להקים מדינה המקיימת תרבות ומשטר דמוקרטי ואשר אופייה ותרבותה הלאומיים יהיו מושתתים על היהדות. הם לא ראו בשני עקרונות אלו, עקרונות סותרים; אדרבה, על- פי מגילת העצמאות הם נובעים זה מזה: "מדינת ישראל... תהא מושתתת על יסודות החירות, הצדק והשלום – לאור חזונם של נביאי ישראל". סקרי דעת קהל ומחקרים מלמדים שהגדרה זו מקובלת על רובו המכריע של הציבור הישראלי, אולם לצד הסכמה זו מתקיים בישראל דיון ארוך שנים כיצד על היסודות היהודיים והדמוקרטיים לבוא לידי ביטוי.
</p>
```

File: wiki/1/1.1.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+<p dir="rtl">בשנת ,1992 עם חקיקתם של חוקי היסוד בהם הופיעה לראשונה ההגדרה "מדינה יהודית ודמוקרטית" באופן מפורש,
+החל דיון ציבורי ביחס למשמעות המושג. שלושה משופטי בית- המשפט העליון הביעו בנושא זה עמדה מנומקת:
+</p>
+
+<p dir="rtl"><b>השופט חיים כהן (1911-2002) </b>סבר כי יש למצוא את שורשי הדמוקרטיה ביהדות ובמשפט לעצב ולבנות העברי; הוא ראה בצמד המילים "יהודית" ו"-דמוקרטית," מושג אחד וגרס כי יש "יהדות שהיא דמוקרטית ודמוקרטיה שהיא יהודית".
```
Comment (IgalKolihman) [inline wiki/1/1.1.md:14]: and make the text is still over 120 chars in a line

File: wiki/1/1.2.md [hunk 7]
```diff
@@ -0,0 +1,181 @@
...
+</p>
+</h1>
+
+<p dir="rtl">
+ביסוד הדמוקרטיה עומדת ההבנה כי האדם הוא יצור תבוני וריבוני, ולכן, לכל אדם הזכות והחובה להיות אחראי לגורלו החברתי והפוליטי ולהשפיע על עיצובה של המדינה בה הוא חי. בהיסטוריה העולמית ישנן דוגמאות למספר דגמים של משטרים דמוקרטיים, על כן אין צורה אחת "נכונה" של דמוקרטיה; אולם העיקרון הבסיס של ריבונות העם הוא המשותף לכל הדמוקרטיות .אופייה הדמוקרטי של כל מדינה, ובכלל זה גם של מדינת ישראל, מבוסס על שתי
+מערכות:
+</p>
+
+<p dir="rtl">
+•     מערכת פורמלית-מוסדית הכוללת תהליכים מוסכמים הקיימים בכל משטר דמוקרטי ובאמצעותם מתממש עקרון ריבונות העם: הכרעת הרוב, קיומה של אופוזיציה ובחירות חופשיות ושוויוניות.
```
Comment (IgalKolihman) [inline wiki/1/1.2.md:13]: better if it was `<li>` and not `•     `

File: wiki/1/1.1.md [hunk 8]
```diff
@@ -0,0 +1,52 @@
+﻿<h1>
+<p dir="rtl">
+1.1 פתיחה: מדינת ישראל כמדינה יהודית ודמוקרטית
+</p>
```
Comment (IgalKolihman) [inline wiki/1/1.1.md:4]: remove the `<p>` tags for content inside `h#` titles

File: wiki/1/1.2.md [hunk 9]
```diff
@@ -0,0 +1,181 @@
...
+</p>
+
+<p dir="rtl">
+במדינת ישראל מתקיימת הן המערכת הפורמלית (מוסדות שלטון דמוקרטיים, בחירות, חילופי שלטון) והן המערכת התרבותית (שוויון, אפליה מתקנת, מדיניות רווחה) . אחד מן ההיבטים הפורמליים החשובים ביותר הוא עקרון הפרדת הרשויות. משמעו, שהמשטר הדמוקרטי מורכב משלוש רשויות נפרדות: הרשות המחוקקת האחראית על חקיקתם של חוקים; הרשות המבצעת האחראית על ביצוע החוק ועל ניהול מוסדות המדינה; והרשות השופטת המפקחת על ביצוע החוק ונותנת סעד לאזרח אל מול החוק ואל מולם של אזרחים אחרים. כאשר שלושת הרשויות נפרדות, הן מאזנות ובולמות האחת את השנייה והכוח השלטוני מבוזר בין שלושתן. מצב בו אין הפרדת רשויות, והחוק ,המשפט והביצוע מרוכזים בידי אדם אחד, עשוי להביא לשחיתות שלטונית, לפגיעה בחירות, לאי-צדק ,למצב בו קבוצה קטנה משתלטת על עמדות הכוח במדינה או למצב
+בו לא נשמרות זכויות המיעוט.
+</p>
+
+<h2>
+<p dir="rtl">
+1.2.1 הרשות המחוקקת: כנסת ישראל</p>
```
Comment (IgalKolihman) [inline wiki/1/1.2.md:28]: when you upload your changes, review the looks of it in the preview. all the titles do not present correctly.

File: wiki/1/1.2.md [hunk 10]
```diff
@@ -0,0 +1,181 @@
...
+המפלגה הגדולה ביותר בממשלה, הופך להיות לראש הממשלה. הכנסת מחולקת לשתי קבוצות: קואליציה המורכבת מחברי כל המפלגות השותפות בממשלה ותפקידה לגבות בכנסת את צעדי הממשלה ולנסות להגן עליהם; ואופוזיציה המורכבת מחברי כל המפלגות שאינן שותפות בממשלה ותפקידה לנסות ולהציע בפני הציבור אלטרנטיבה לפעולות הממשלה ולהוביל את הביקורת על צעדיה. במצב תקין, הקואליציה גדולה יותר מן האופוזיציה וכך יש רוב בכנסת השומר על יציבות הממשלה ומבטיח את יכולתה להביא את מדיניותה
+לכדי ביצוע. מצב בו האופוזיציה גדולה יותר מן הקואליציה, אינו מאפשר לממשלה לפעול ויוביל באופן טבעי לכינונה של ממשלה חדשה ובמקרים מסוימים גם לפיזור הכנסת ולעריכת בחירות חדשות. לכנסת שני תפקידים מרכזיים: חקיקת חוקים ובקרה על עבודת הממשלה. חברי הכנסת רשאים לחוקק כל חוק ובפרט
+שהוא "חוקתי," קרי, אינו סותר את חוקי היסוד ואינו סותר את הגדרתה של ישראל כמדינה יהודית ודמוקרטית. הבקרה על עבודת הממשלה נעשית באמצעות היכולת להצביע בה" אי-אמון" ובכך לגרום לנפילתה, באמצעות היכולת לשאול "שאילתות" או להעלות נושאים לדיון ובאמצעות החובה לאשר במליאה צעדים מסוימים של הממשלה כגון ההחלטה על תקציב המדינה, חתימת חוזה שלום, יציאה למלחמה ועוד .
```
Comment (IgalKolihman) [inline wiki/1/1.2.md:35]: this is an extremely long paragraph, separate into smaller paragraphs.

File: wiki/1/1.3.md [hunk 11]
```diff
@@ -0,0 +1,74 @@
...
+</h2>
+
+<p dir="rtl">
+עם הקמתה ביקשה מדינת ישראל לאמץ לעצמה סמל רשמי. מועצת המדינה הזמנית מינתה ועדת סמלים מיוחדת, אשר פנתה אל תושבי המדינה והזמינה אותם להציע את הצעותיהם. מתוך מאות הצעות שהוגשו לוועדה, נבחרה בסיכומו של התהליך הצעתם של המעצבים גבריאל ומקסים שמיר. הסמל אותו עיצבו שוזר יחדיו שלושה מרכיבים:
+מנורה בת שבעה קנים, ענפי זית והשם "ישראל". המנורה מייצגת את עברו הלאומי של העם ואת מאבקו לחירות. צורת המנורה המופיעה בסמל, מבוססת על תבליט המופיע על שער טיטוס ברומא בו מתוארים כלי המקדש שנלקחו בשבי עם חורבן בית-המקדש השני.
+\*\* לבחירה זו משמעות סמלית – אותה מנורה שציינה את היציאה לגלות, מבטאת כיום את חזרתם של היהודים למדינתם הריבונית. ענפי הזית מסמלים את העתיד ואת השאיפה לשלום. סמל המדינה קשור בבסיסו גם אל חזון זכריה שבאחת מנבואותיו נגלה אליו מחזה של מנורה (זכריה ד ,ב-ג).
+</p>
+
+<h6 dir="rtl">
+\* טבח כפר קאסם נידון בפרק נפרד בהמשך.
```
Comment (IgalKolihman) [inline wiki/1/1.3.md:24]: replace `\*` with something else that will still gives the same meaning. maybe replace to `[*]`

**Review by IgalKolihman — CHANGES_REQUESTED** [review]

File: wiki/1/1.1.md (hunk 6 above)
Comment (IgalKolihman) [reply wiki/1/1.1.md:14]: forget about 120 chars in a line.

**Pull request merged**